docs/source/benchmarking.rst

```diff
@@ -71,73 +90,84 @@ Aggregate across simulated communities (when dataset index is `not` unique)
     df = pd.concat(dfs)
     df.to_csv("benchmarks.tsv", sep='\t', index=True, header=True)
 
-Downloading Test Datasets
-=========================
-
-Using the built-in ``autometa`` module
---------------------------------------
-
-Autometa is packaged with a built-in module that allows any user to download any of the available test datasets.
-To use these utilities simply run the command ``autometa-download-dataset``.
+Benchmark classification
+^^^^^^^^^^^^^^^^^^^^^^^^
 
-For example, to download all of the simulated communities reference binning/taxonomy assignments as well as the Autometa
-v2.0 binning/taxonomy predictions:
+Previously we benchmarked the clustering step. Now we are going to benchmark the taxon profiling step. The only difference is that the ``--classification`` flag is provide instead of the ``--clustering`` flag.
 
 .. code:: bash
 
-    # Note: community is the test dataset that was used for clustering or classification. e.g.
-    # choices: 78Mbp,156Mbp,312Mbp,625Mbp,1250Mbp,2500Mbp,5000Mbp,10000Mbp
-    community_sizes=(78Mbp 156Mbp 312Mbp 625Mbp 1250Mbp 2500Mbp 5000Mbp 10000Mbp)
-
-    autometa-download-dataset \
-    --community-type simulated \
-    --community-sizes ${community_sizes[@]} \
-    --file-names reference_assignments.tsv.gz binning.tsv.gz taxonomy.tsv.gz \
-    --dir-path simulated
+    autometa-benchmark \
+        --benchmark classification \
+        --predictions taxonomy.tsv.gz \
+        --reference simulated/${community_size}/reference_assignments.tsv.gz \
+        --output-wide ${community_size}.classification_benchmarks.wide.tsv.gz \
+        --output-classification-reports taxa_reports \
+        --ncbi /ncbi
 
+Benchmark clustering-classification
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Using ``gdrive`` via command line
----------------------------------
+This will benchmark the final binning results using the classification metrics, 
+precision, recall and F1-score.
 
-You can download the individual assemblies of different datasests with the help of ``gdown`` using command line.
-If you have installed ``autometa`` using ``conda`` then ``gdown`` should already be installed.
-If not, you can install it using ``conda install -c conda-forge gdown`` or ``pip install gdown``.
+.. note::
+    
+    First, F1-scores are determined for each reference genome across all recovered genome bins (specifically for clusters containing any of the reference genome's contigs). Then, each reference genome's F1-score is selected corresponding to the respective cluster with the largest set of the respective reference genome's contigs (by length in bp).
 
-Example for the 78Mbp simulated community
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. code:: bash
+    
+    autometa-download-dataset \
+        --community-type simulated \
+        --community-sizes 78Mbp \
+        --file-names binning.tsv.gz\
```

Comment:
We should also download the `reference_assignments.tsv.gz` here so this section can be performed standalone

```suggestion
        --file-names binning.tsv.gz,reference_assignments.tsv.gz \
```